components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/token/TokenRevocationNotifier.java

```diff
@@ -46,6 +48,7 @@
      * @param realTimeNotifierProperties
      * @param persistentNotifierProperties
      */
-    void init(Properties realTimeNotifierProperties, Properties persistentNotifierProperties);
+    void init(Properties realTimeNotifierProperties, Properties persistentNotifierProperties)
+            throws APIManagementException;
```

Comment:
Is it correct to throw an APIManagementException here? Isn't there one that matches the current package?

Reply:
There is not a exception defined in the current package. Also, there is no exception class for token revocation in org.wso2.carbon.apimgt.impl.token the package which TokenRevocationNotifier interface is defined.

Reply:
Is it ok to use the notifier exception in the impl? @malinthaprasan

Reply:
@malinthaprasan ping!